Example/Gemini/ViewControllers/CircleRotationViewController.swift

```diff
@@ -50,8 +50,8 @@ final class CircleRotationViewController: UIViewController {
         }
 
         // Switch navigation bar hidden
-        navigationController?.setNavigationBarHidden(true, animated: false)
-        view.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(toggleNavigationBarHidden(_:))))
+        navigationController?.setNavigationBarHidden(false, animated: false)
+        //        view.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(toggleNavigationBarHidden(_:))))
```

Comment:
We remove gesture recognizer to make sure it doesn't intercept our cell selection.